web/modules/custom/joinup_seo/src/Plugin/simple_sitemap/UrlGenerator/JoinupUrlGeneratorBase.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\joinup_seo\Plugin\simple_sitemap\UrlGenerator;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Entity\Query\QueryInterface;
+use Drupal\Core\Language\LanguageManagerInterface;
+use Drupal\joinup_core\JoinupRelationManagerInterface;
+use Drupal\simple_sitemap\EntityHelper;
+use Drupal\simple_sitemap\Logger;
+use Drupal\simple_sitemap\Plugin\simple_sitemap\UrlGenerator\EntityUrlGenerator;
+use Drupal\simple_sitemap\Plugin\simple_sitemap\UrlGenerator\UrlGeneratorManager;
+use Drupal\simple_sitemap\Simplesitemap;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Base class for Joinup Url Generators.
+ *
+ * @package Drupal\joinup_seo\Plugin\simple_sitemap\UrlGenerator
```

Comment:
`@package` is inserted automatically by PHPStorm and is worthless because it just is a duplication of the namespace. It is not allowed according to the Drupal coding standards and should always be removed.